src/main/java/sage/exceptions/MissingParameterException.java

```diff
@@ -0,0 +1,26 @@
+package sage.exceptions;
+
+public class MissingParameterException extends Exception {
+    /**
+     * Exception which returns an error message when there are missing parameter(s) in task operations.
+     *
+     * @return An error message when user provides incomplete/missing parameters in task operations
+     */
+    public String missingParamDesc() {
+        return "Task description parameter is missing";
+    }
+
+    public String missingParamStart() {
+        return "Task from parameter is missing";
```

Comment:
```suggestion
        return "Task \"from\" parameter is missing";
```
Increase clarity to user